### PR TITLE
chore: add AUTHORS and OWNERS files

### PR DESCRIPTION
**Pull request by lance**
Preparation for the move to knative-sandbox.
See: https://github.com/knative/community/issues/613

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

**Comment by lance** [comment]
I have no idea why integration tests are failing. I suspect it has something to do with cached image layers, but it's not obvious. Locally I have `docker prune`d everything, explicitly deleted the go builders and buildpacks, run both `pack build ...` and `kn func build` and have not been able to reproduce this. /cc @jrangelramos

File: AUTHORS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This is the list of func authors for copyright purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder.  To see the full list
+# of contributors, see the revision history in source control.
+Red Hat, Inc.
+
```
Comment (zroubalik) [inline AUTHORS:7]: Is the list of authors intentionally empty?

Comment (lance) [reply AUTHORS:7]: It include "Red Hat, Inc." on line 6. All contributors to date have done so as an employee of Red Hat.

**Review by lkingland — APPROVED** [review]

**Review by zroubalik — APPROVED** [review]

**Comment by lance** [comment]
@jrangelramos do you see any reason why this would be failing integration tests? I've only added a couple of text files to the repo, but it consistently fails in CI. I haven't been able to replicate this locally.

**Review by lkingland — APPROVED** [review]

**Review by matejvasek — APPROVED** [review]

**Pull request merged**